pkgs/development/tools/cosmoc/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, lib, cosmopolitan }:
+
+stdenv.mkDerivation {
+  name = "cosmoc";
```

Comment:
Please use pname and version. version can probably be inherited from cosmopolitan.